modules/nw-about-multicast.adoc

```diff
@@ -14,14 +14,23 @@ discovery and not a high-bandwidth solution.
 ====
 
 Multicast traffic between {product-title} Pods is disabled by default. If you
-are using the OpenShiftSDN network plug-in, you can enable multicast on a
+are using the OpenShift SDN network plug-in, you can enable multicast on a
 per-project basis.
 
-When using the OpenShiftSDN network plug-in:
+When using the OpenShift SDN network plug-in in `networkpolicy` isolation mode:
 
 * Multicast packets sent by a Pod will be delivered to all other Pods in the
-project, regardless of `NetworkPolicy` objects. Pods may be able to communicate
+project, regardless of NetworkPolicy objects. Pods may be able to communicate
 over multicast even when they cannot communicate over unicast.
 * Multicast packets sent by a Pod in one project will never be delivered to Pods
-in any other project, even if there are `NetworkPolicy` objects allowing
+in any other project, even if there are NetworkPolicy objects allowing
 communication between the projects.
+
+When using the OpenShift SDN network plug-in in `multitenant` isolation mode,
+multicast packets sent by a Pod in one project will be delivered to Pods in
+another project only if the following conditions are true:
+
+* Each project must be joined together to allow multicast packets to traverse
+between Pods.
+* Each project must have multicast enabled.
+
```

Comment:
Below two test cases passed in v4.1:
1. multicast with networkpolicy mode 
2. the multicast packets can traverse between Pods which in two joined together projects